css/settings.scss

```diff
@@ -76,6 +76,8 @@ $layout-max-width: 1080px;
  * SPACES && SIZES
 */
 // $space-1: 8px;
+$nav-height: 60px;
```

Comment:
What I've done for the grid is to keep the variables in the components. That's why the grid-item file must be the first one of the grid components.

Having them outside is better to avoid to order the components but at the same time, those variables are really specific. Which way do you want to go?